app/Product/Aspect/ExampleProductAspect.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Example\Project\Product\Aspect;
+
+use Go\Aop\Aspect;
```

Comment:
Go is super magic to me I love that I can play around with it in this repo and not kill my clients with my inexperience.